internal/test/integration/rpm_test.go

```diff
@@ -420,34 +404,40 @@ func (r *RpmSuite) TestRpmRepositoryVersionModuleStreamsList() {
 	singleList, err = r.tangy.RpmRepositoryVersionModuleStreamsList(context.Background(), []string{*firstVersionHref}, tangy.ModuleStreamListFilters{RpmNames: []string{"banana"}}, "")
 	require.NoError(r.T(), err)
 	assert.Empty(r.T(), singleList)
+
+	// Test no duplicated module names
+	singleList, err = r.tangy.RpmRepositoryVersionModuleStreamsList(context.Background(), []string{*firstVersionHref, *secondVersionHref}, tangy.ModuleStreamListFilters{RpmNames: []string{"walrus", "kangaroo"}}, "name DESC")
+	require.NoError(r.T(), err)
+	assert.Equal(r.T(), singleList[0].Name, "walrus")
+	assert.Len(r.T(), singleList, 1)
```

Comment:
test is failling because this list contains 3 items, which i **think** is correct from the fixtures? ([v1](https://github.com/content-services/fixtures/blob/main/fixtures/yum/comps-modules/v1/repodata/modules.yaml), [v2)](https://github.com/content-services/fixtures/blob/main/fixtures/yum/comps-modules/v2/repodata/modules.yaml)